openstack_controller/datadog_checks/openstack_controller/config.py

```diff
@@ -47,6 +47,7 @@ def __init__(self, logger, config):
         self.ironic_microversion = config.ironic_microversion
         self.endpoint_interface = config.endpoint_interface
         self.endpoint_region_id = config.endpoint_region_id
+        self.paginated_limit = instance.get("paginated_limit", None)
```

Comment:
Why not `config.paginated_limit`? This way we would save an argument

Reply:
If we use config.paginated_limit, the default value will be 1000 instead of None

Reply:
I guess we have to align those values

Reply:
We could instead remove the default in the conf.yaml and have the default be `None`

Reply:
It's ok for me